app/controllers/user_filters_controller.rb

```diff
@@ -5,12 +5,20 @@ class UserFiltersController < ApplicationController
   before_action :authenticate_user!
   before_action :check_role
 
+  # Maximum number of allowed user filters for the Advanced Filter.
+  MAX_SAVED_FILTERS = 100
```

Comment:
make this 150, and put it as a config variable in https://github.com/SaraAlert/SaraAlert/blob/master/config/sara/sara.yml

Reply:
Done here: 37786fb